drivers/hmis/spec/system/hmis/assessment_definitions_spec.rb

```diff
@@ -181,17 +179,17 @@ def select_member(client)
     end
 
     def unlock_household_assessment
-      find('button', text: 'Unlock Assessment').trigger('click')
+      click_button 'Unlock Assessment'
       assert_text 'Save & Submit' # Unlock succeeded
     end
 
     def submit_household_assessment
-      find('button', text: 'Save & Submit').trigger('click') # note: using click_button fails on CI (#6619)
+      click_button 'Save & Submit'
```

Comment:
Can easily reproduce the fix here. The `SaveSlide` component at the bottom of the assessment was sliding in, which was causing click_button not to work. Now that the transitions are disabled, it no longer slides in, so this works